### DISCUSSION TITLE
What's the procedure/process to be member of KEDA and not just contributor?

### DISCUSSION THREAD

**Discussion by Ritikaa96** [kedacore/governance#29] [Q&A] [answered]
I have seen `member` tag on different contributors.
While there is a document for maintainers and their roles, I could not find the role responsibilities and the process for Members.
Is there any document? or even the role as Member? 
If not it might be beneficial for the community to redefine the roles and hierarchy.

**Comment by tomkerkhove** [comment, marked as answer]
We don't have any strict procedures or process for members of KEDA, the general rule is that they are a maintainer or a lead of a component.

For example, you'll see @arschles who is a member because he's leading the HTTP add-on or people from Microsoft/Red Hat since they were part of the early prototype.

However, being a member doesn't mean anything though. Is there a reason why you are asking?

**Reply by Ritikaa96** [reply]
Hi, @tomkerkhove  Got it . Thanks. Seeing the tag, curious whether we have something similar to the Member role of other CNCF projects like k8s, envoy, etc as it's not mentioned as part of governance. Thanks for clarifying.

**Reply by arschles** [reply]
@Ritikaa96 as @tomkerkhove said, I am leading the [HTTP Addon](https://github.com/kedacore/http-add-on) part of this project. We are looking for contributors there, especially those who might be interested in growing to a maintainership role. Let me know if you're interested. You can also send me a direct message on the Kubernetes slack if you'd rather. I'm `arschles` there.

**Reply by Ritikaa96** [reply]
Hi, @arschles thank you, I'm surely interested in contributing more. I'm investigating the HTTP Addon too so you might hear from me soon.

**Reply by arschles** [reply]
wonderful :)

**Reply by tomkerkhove** [reply]
I have created https://github.com/kedacore/governance/issues/30 to see what we can do
